### PR TITLE
Highlight the current utam PO.

### PR DESCRIPTION
**Pull request by qiaoyang2008**

### CODE REVIEW

File: app/renderer/actions/Inspector.js [hunk 1]
```diff
@@ -430,6 +431,32 @@ export function searchForElement (strategy, selector) {
...
+
+export function resetSearchForPORootElement () {
+  return (dispatch) => {
+    dispatch({type: SEARCHING_FOR_ROOT_ELEMENTS});
+  };
+}
+
+export function searchForPORootElement (strategy, selector, po) {
+  return async (dispatch, getState) => {
+    //dispatch({type: SEARCHING_FOR_ROOT_ELEMENTS});
```
Comment (helenren) [inline app/renderer/actions/Inspector.js]: nit: pls remove the comment out code

File: package.json [hunk 2]
```diff
@@ -153,6 +153,7 @@
     "i18next-http-backend": "1.4.4",
     "i18next-localstorage-backend": "3.1.3",
     "i18next-node-fs-backend": "2.1.3",
+    "is-extendable": "^1.0.1",
```
Comment (helenren) [inline package.json:156]: seems no place use this, right? If no, pls remove this no use decency.

**Review by helenren — APPROVED** [review]
Overall, the changes LGTM, just two minor comments, pls address them.

File: package.json (hunk 2 above)
Comment (qiaoyang2008) [reply package.json:156]: This is need to get the project build successfully(npm run build).

File: app/renderer/actions/Inspector.js (hunk 1 above)
Comment (qiaoyang2008) [reply app/renderer/actions/Inspector.js]: will remove it.

**Pull request merged**